applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -26,12 +60,12 @@ def _ValidateSettings(self, settings):
                 "materials_filename": "unknown_materials.json"
             },
             "formulation": {
-                "element_type": "imcompressible_potential"
+                "element_type": "incompressible"
             },
             "maximum_iterations": 10,
             "echo_level": 0,
-            "relative_tolerance": 1e-5,
-            "absolute_tolerance": 1e-9,
+            "relative_tolerance": 1e-12,
```

Comment:
Maybe is to much to be the default?

Reply:
For now in 2d it is giving very nice results, most probably we will need to change it when we go 3d but I would leave it as it is for now if it is ok for you.